python/test/unit/mesh/test_mesh.py

```diff
@@ -205,6 +205,104 @@ def test_mesh_construction_pygmsh():
     assert mesh.topology.dim == 2
 
 
+def test_mesh_construction_gmsh():
+    import gmsh
+
+    # map from gmsh to type that is also used by meshio
```

Comment:
Please make first word capitalised in comments, we follow pep8 standard, https://www.python.org/dev/peps/pep-0008/#comments